modules/evm/src/lib.rs

```diff
@@ -1774,6 +1776,10 @@ impl<T: Config> EVMTrait<T::AccountId> for Pallet<T> {
 	fn set_origin(origin: T::AccountId) {
 		ExtrinsicOrigin::<T>::set(Some(origin));
 	}
+
+	fn clear_origin() {
+		ExtrinsicOrigin::<T>::kill();
+	}
```

Comment:
It was removed here.
https://github.com/AcalaNetwork/Acala/blob/3c25a51339bfda1391dd9c541a8f9e70f1187812/modules/evm/src/lib.rs#L1914

Reply:
remove it now. previous I'm using it in testcase to manual clear origin.